components/ide/code/leeway.Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN sudo apt-get update \
     && sudo apt-get clean -y \
     && rm -rf /var/lib/apt/lists/*
 
-ENV GP_CODE_COMMIT ae8dc44299153ca820ac9e13cc9def5bb6baafd1
+ENV GP_CODE_COMMIT 334e0610dd1bb73487812b417b113d6a2b96e1b1
```

Comment:
**issue**: The Extend Workspace Timeout throws an error. Is this expected? If yes, do you think it would make sense to add a small hyperlink in the notification item to also link to `/upgrade-subscription`?

> Cannot extend workspace timeout: Error: enterprise license required

Reply:
yes, it is expected. I need to check whether hyperlinks are supported in notifications.